src/lib/components/Hero.svelte

```diff
@@ -67,4 +53,52 @@
 		background-size: cover;
 		background-position: center;
 	}
+
+	h1 {
+		line-height: 0.8;
+		@media (max-width: 1282px) {
+			line-height: 1.1;
+		}
+	}
+
+	a {
+		font-weight: 600;
+	}
+
+	img {
+		@media (max-width: 1282px) {
+			width: 75%;
+			margin-inline: auto;
+		}
+		@media (max-width: 768px) {
+			width: 100%;
+		}
+	}
+
+	.hello {
+		align-items: center;
+	}
+
+	.fuck {
```

Comment:
lol malo uce, just this one change please, in case we're showcasing to the public.

Reply:
Hahahahhaa my bad!! I thought I changed that before, sorry about that

Reply:
Fixed! :D